### PR TITLE
Docstring bootstrap plot

### PR DESCRIPTION
**Pull request by aydevosotros**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [x] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
Docstring for "pandas.plotting._misc.bootstrap_plot" correct. :)

```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

### CODE REVIEW

File: pandas/plotting/_misc.py [hunk 1]
```diff
@@ -323,20 +323,51 @@ def f(t):
...
     Returns
     -------
-    fig: matplotlib figure
+
+    fig : matplotlib.figure.Figure
+        Matplotlib figure
+
+    See Also
+    --------
+
```
Comment (TomAugspurger) [inline pandas/plotting/_misc.py]: Remove this newline.

File: pandas/plotting/_misc.py [hunk 2]
```diff
@@ -323,20 +323,51 @@ def f(t):
...
+        greater or equal than the length of the `series`.
+    samples : int, default 500
+        Number of times the bootstrap procedure is performed.
+    kwds : keywords
+        Options to pass to matplotlib plotting method.
 
     Returns
     -------
-    fig: matplotlib figure
+
```
Comment (TomAugspurger) [inline pandas/plotting/_misc.py]: Remove newline

File: pandas/plotting/_misc.py [hunk 3]
```diff
@@ -323,20 +323,51 @@ def f(t):
...
 
     Parameters
     ----------
-    series: Time series
-    fig: matplotlib figure object, optional
-    size: number of data points to consider during each sampling
-    samples: number of times the bootstrap procedure is performed
-    kwds: optional keyword arguments for plotting commands, must be accepted
-        by both hist and plot
+
```
Comment (TomAugspurger) [inline pandas/plotting/_misc.py]: Remove newline. (should go right below the `----` line)

File: pandas/plotting/_misc.py [hunk 4]
```diff
@@ -323,20 +323,51 @@ def f(t):
...
+    pandas.Series.plot : Basic plotting for Series objects.
+
+    Examples
+    --------
+
+    .. plot::
+            :context: close-figs
+
+            >>> from random import uniform
+            >>> s = pd.Series([uniform(0, 1) for _ in range(100)])
```
Comment (TomAugspurger) [inline pandas/plotting/_misc.py]: Numpy is imported.

s = pd.Series(np.random.uniform(size=100))

**Comment by aydevosotros** [comment]
Already fixed the wrong new lines

File: pandas/plotting/_misc.py [hunk 5]
```diff
@@ -323,20 +323,49 @@ def f(t):
 
 
 def bootstrap_plot(series, fig=None, size=50, samples=500, **kwds):
-    """Bootstrap plot.
+    """
+    Bootstrap plot on mean, median and mid-range statistics.
+
+    The bootstrap plot is used to estimate the uncertainty of a statistic
```
Comment (jreback) [inline pandas/plotting/_misc.py]: can we add a wikipedia reference in Reference

File: pandas/plotting/_misc.py [hunk 6]
```diff
@@ -323,20 +323,49 @@ def f(t):
...
+        Pandas Series from where to get the samplings for the bootstrapping.
+    fig : matplotlib.figure.Figure, default None
+        If given, it will use the `fig` reference for plotting instead of
+        creating a new one with default parameters.
+    size : int, default 50
+        Number of data points to consider during each sampling. It must be
+        greater or equal than the length of the `series`.
+    samples : int, default 500
+        Number of times the bootstrap procedure is performed.
+    kwds : keywords
```
Comment (jreback) [inline pandas/plotting/_misc.py]: dict, I think this needs to be **kwds. @TomAugspurger

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/plotting/_misc.py (hunk 6 above)
Comment (TomAugspurger) [reply pandas/plotting/_misc.py]: Yeah we're doing `**kwds` with no type (it's not really a dict, but an unpacked dict).

Sorry @aydevosotros you can ignore the validation script here. Haven't updated it yet.

File: pandas/plotting/_misc.py (hunk 3 above)
Comment (TomAugspurger) [reply pandas/plotting/_misc.py]: I think this line still needs to be removed.

**Comment by TomAugspurger** [comment]
I see a couple warnings locally with `./make.py html --single=pandas.plotting.bootstrap_plot` from the `doc` directory. Can you check that @aydevosotros? along with the extra newline and the `**kwds`.

```
reading sources... [100%] index
/Users/taugspurger/sandbox/pandas-ip/pandas/pandas/plotting/_misc.py:docstring of pandas.plotting.bootstrap_plot:53: WARNING: Explicit markup ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
pickling environment... done
checking consistency... /Users/taugspurger/sandbox/pandas-ip/pandas/doc/source/generated_single/pandas.plotting.bootstrap_plot.rst:52: WARNING: Citation [R1] is not referenced.
```

**Comment by TomAugspurger** [comment]
Those look good. I think just remove the Notes section and footnote. You can include the wikipedia link in the extended summary.

**Comment by aydevosotros** [comment]
@TomAugspurger I think now fits well. Just so sorry for that much time taken because of my inexperience. Thank you :)

**Comment by TomAugspurger** [comment]
Thanks @aydevosotros.

**Pull request merged**